### PR TITLE
riemann-tools: 0.2.13 -> 0.2.14, add more config options

### PR DESCRIPTION
**Pull request by DerTim1**
###### Motivation for this change
Update riemann-tools (minor version).
Add more NixOS config options (attributes and commandSwitches) to be able to customize the monitoring client.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: nixos/modules/services/monitoring/riemann-tools.nix [hunk 1]
```diff
@@ -34,8 +37,24 @@ in {
...
       };
+      attributes = mkOption {
+        type = types.listOf types.string;
+        default = [];
+        description = ''
+          A list of attributes and their values (name=value) added to the events.
+        '';
+        example = ["myattribute=42"];
+      };
+      commandSwitches = mkOption {
```
Comment (srhb) [inline nixos/modules/services/monitoring/riemann-tools.nix]: There's some precedence for calling this `extraArgs`, so I think we should stick with that.

File: nixos/modules/services/monitoring/riemann-tools.nix [hunk 2]
```diff
@@ -34,8 +37,24 @@ in {
           Address of the host riemann node. Defaults to localhost.
         '';
       };
+      attributes = mkOption {
```
Comment (srhb) [inline nixos/modules/services/monitoring/riemann-tools.nix]: I feel like this doesn't add a lot of value over `commandSwitches` (which I would prefer be called `extraArgs`) -- can we scrap it? Keeping NixOS modules small and simple is good for evaluation time and maintainability :)

**Comment by DerTim1** [comment]
Thanks for your advice @srhb 
I renamed the config option and removed the special option for attributes.

**Review by srhb — APPROVED** [review]

**Comment by mmahut** [comment]
Updated in https://github.com/NixOS/nixpkgs/pull/65691.

**Comment by DerTim1** [comment]
@mmahut This PR also adds more config options for NixOS, so commit 9c07699e5b394e162ce3b6d46ca37573d2978736 should be merged.

**Comment by mmahut** [comment]
@DerTim1 do you mind opening a new PR for that commit please?

**Comment by DerTim1** [comment]
@mmahut Can you cherry pick the commit or do a rebase? Unfortunately I'm busy with other stuff at the moment...

**Comment by DerTim1** [comment]
@mmahut Ok, spend 5 minutes and created a new PR. See #66606.

**Pull request closed without merging**